content/en/releases/OWNERS

```diff
@@ -9,7 +9,9 @@ reviewers:
 
 approvers:
   - sig-docs-en-owners
+  - sig-release-leads
   - release-engineering-approvers
 
 labels:
-- sig/release
\ No newline at end of file
+- sig/release
+- area/release-eng
```

Comment:
We might need to add this label to the repo as well(?)

Reply:
PR opened for the `area/release-eng` label here: https://github.com/kubernetes/test-infra/pull/22272